Dockerfile

```diff
@@ -1,8 +1,8 @@
 FROM golang:1.10.1-alpine3.7 as go-builder
 
 ARG PROJECT_NAME=pravega-operator
-ARG REPO_PATH=github.com/pravega/$PROJECT_NAME
-ARG BUILD_PATH=${REPO_PATH}/cmd/${PROJECT_NAME}
+ARG REPO_PATH=github.com/pravega/${PROJECT_NAME}
+ARG BUILD_PATH=${REPO_PATH}/cmd/manager
```

Comment:
why is this not the project name any longer?

Reply:
That's one of the changes in the new SDK. See [Copy changes from main.go](https://github.com/operator-framework/operator-sdk/blob/master/doc/migration/v0.1.0-migration-guide.md#copy-changes-from-maingo).

> The main function for a `v0.1.0` operator in `cmd/manager/main.go` sets up the Manager which registers the custom resources and starts all the controllers.

> `operator-sdk` now expects `cmd/manager/main.go` to be present in Go operator projects.